lib/components/user/nav-login-button-auth0.js

```diff
@@ -6,18 +6,28 @@ import NavLoginButton from './nav-login-button'
 /**
  * This component wraps NavLoginButton with Auth0 information.
  */
-export default function NavLoginButtonAuth0 (props) {
+const NavLoginButtonAuth0 = ({
+  className,
+  id,
+  links,
+  style
+}) => {
   const { isAuthenticated, login, logout, user } = useAuth()
 
   const handleLogin = () => login({ appState: {returnTo: window.location.href} })
   const handleLogout = () => logout({ returnTo: window.location.href })
```

Comment:
I really don't think we want to run the risk of upsetting Auth0 by potentially returning the user to a bad URL (this could happen if we use href and somehow down the road are not using hash routing or if the href has a path set for some other reason).